tests/foreman/cli/test_errata.py

```diff
@@ -565,12 +1430,36 @@ def test_positive_list_filter_by_org_label(self):
         1. erratum list --organization-label=<org_label>
 
         @Assert: Errata is filtered by Org label.
-
-        @caseautomation: notautomated
-
         """
-
-    @stubbed()
+        org_erratum_list = Erratum.list({
+            'organization-label': self.org['label'],
+            'per-page': 1000
+        })
+        org_errata_ids = {
+            errata['errata-id']
+            for errata in org_erratum_list
+        }
+        errata_org_erratum_list = Erratum.list({
+            'organization-label': self.errata_org['label'],
+            'per-page': 1000
+        })
+        errata_org_errata_ids = {
+            errata['errata-id']
+            for errata in errata_org_erratum_list
+        }
+        self.assertEqual(len(org_errata_ids), self.org_product_erratum_count)
```

Comment:
It seem same code as the one starting on line 1370. Same method extraction could apply on this code, similar to the one I proposed.